other/tor.md

```diff
@@ -1,5 +1,11 @@
 # Tor hidden service peers
 
+*It's recommended that Tor peers use Tor's [LongLivedPorts] (21, 22, 706,
```

Comment:
Another thing I am not entirely happy with is how I repat "Tor" twice here, does anyone have better wording suggestions?

Reply:
>Tor peers

this is already page for Tor peers so you can skip that and just have "peers", I don't think that's going to be confusing.

Reply:
Fixed, thanks :)